Update note on log status remapper (#25271)
DataDog/documentation: content/en/logs/log_configuration/processors.md

* Strings beginning with **o** or **s**, or matching **OK** or **Success** (case-insensitive) map to **OK**
* All others map to **info (6)**

> **Note**: Note: If multiple log status remapper processors are applied to a log within a pipeline, only the first one in the pipeline’s order is considered. Additionally, across all pipelines that match the log, only the first status remapper encountered (from all applicable pipelines) will be applied.

Reviewer: ```suggestion
**Note**: If multiple log status remapper processors are applied to a log within a pipeline, only the first one in the pipeline's order is considered. Additionally, for all pipelines that match the log, only the first status remapper encountered (from all applicable pipelines) is applied.
```
